docs/backends/generic-mysql.rst

```diff
@@ -136,7 +136,11 @@ Use the InnoDB READ-COMMITTED transaction isolation level. Default: yes.
 ``gmysql-ssl``
 ^^^^^^^^^^^^^^^^^^
 
-Send the CLIENT_SSL capability flag to the server. SSL support is announced by the server via CLIENT_SSL and is enabled if the client returns the same capability. Default: no.
+.. versionchanged:: 4.9.0
+
+Before 4.9.0: Send the CLIENT_SSL capability flag to the server. SSL support is announced by the server via CLIENT_SSL and is enabled if the client returns the same capability. Default: no.
```

Comment:
`Default: no` wants to be in its own line, after all version things?

Reply:
(I feel like a statement ala "The exact meaning of these flags are MySQL-client and -version dependent" might also be a good C-Y-A line...)